src/util/constants.js

```diff
@@ -12,4 +12,5 @@
 export const INSTANCES = [
   { name: "ICARE", url: "https://skyportal-icare.ijclab.in2p3.fr" },
```

Comment:
Fine for testing, but unlike Michael's testing prototype, we will want these to be a list that the user can add other instances too. Ideally, cache the instance URL + token or something.

Reply:
Created issue #17 for this